tests/test_other.py

```diff
@@ -1323,7 +1327,7 @@ def test_redundant_link(self):
 
     Building.emcc(lib_name, output_filename='libA.so')
 
-    Building.emcc(main_name, ['libA.so'] * 2, output_filename='a.out.js')
+    Building.emcc(main_name, ['libA.so', 'libA.so'], output_filename='a.out.js')
```

Comment:
heh, i don't care either way, but why didn't you like the `* 2` here? :)

Reply:
I'll revert that.  I just though it was too clever for its own good.. especially in test code :)